hs_composite_resource/tests/test_composite_resource.py

```diff
@@ -1660,8 +1660,8 @@ def test_resource_coverage_auto_update(self):
         self.assertEqual(res_coverage.value["units"], "Decimal degrees")
         self.assertEqual(res_coverage.value["northlimit"], 43.6789)
         self.assertEqual(res_coverage.value["eastlimit"], -110.88845678)
-        self.assertEqual(res_coverage.value["southlimit"], 41.98722286030317)
-        self.assertEqual(res_coverage.value["westlimit"], -111.6975629308406)
+        self.assertEqual(res_coverage.value["southlimit"], 41.98722286030319)
```

Comment:
I assume the floating point precision is due to a gdal upgrade

Reply:
yep